src/main/resources/properties/configuration.properties

```diff
@@ -1,3 +1,3 @@
 configuration.profiles=
-configuration.suites=
-configuration.environments=
+configuration.suites=rest_api, web_app, electron
```

Comment:
we do not want to run all suites at once, users should choose which suite to run